apps/frontend/src/modules/events/components/index.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+    EventDefinitionCountableEdge,
+    EventPropertyWithDefinition,
+    EventsQuery,
+    useEventDefinitionsQuery,
+    useEventsLazyQuery,
+    usePropertiesWithDefinitionsLazyQuery,
+} from "@/generated/graphql";
+import { Dialog, DialogContent, DialogTrigger, Header, Spinner, TextInput } from "@/ui";
+import { QuestionIcon, Search } from "@/ui/icons";
+import PeopleIconLg from "@/ui/icons/PeopleIconLg";
+import { Table, TableBody, TableCell, TableHead, TableHeaderCell, TableRow } from "@tremor/react";
+import { format } from "date-fns";
+import { useState } from "react";
+
+export const EventsIndex = () => {
```

Comment:
@afolasope I can see the completed component now. Can you remove the other one? Not sure why you're keeping it since it's no longer needed.